rb/lib/selenium/webdriver/chrome/service.rb

```diff
@@ -35,6 +34,11 @@ class Service < WebDriver::Service
         ERROR
         @shutdown_supported = true
 
+        def self.driver_path=(path)
```

Comment:
I think this can be moved into the base class, which would also solve the issue of none of the other current implementations of `driver_path=` having the String check

Reply:
My reluctance was that it doesn't make sense for the Base class itself to have a driver_path, and if for some reason it was set on the base class that would change any of the subclass values.

Reply:
@titusfortner Setting on the base class wouldn't affect anything -- With class instance variables any class derived from the base class has its own copy of the variable